### PR TITLE
Add `meroxa functions log`

### PR DESCRIPTION
**Pull request by owenthereal**
## Description of change

Add `meroxa functions log`.

Fixes https://github.com/meroxa/cli/issues/244.

## Type of change

- [x]  New feature
- [ ]  Bug fix
- [ ]  Refactor
- [ ]  Documentation

## How was this tested?

- [x]  Unit Tests
- [ ]  Tested in staging

### CODE REVIEW

File: cmd/meroxa/root/functions/logs.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+type functionLogsClient interface {
+	GetFunctionLogs(ctx context.Context, nameOrID string) (*http.Response, error)
+}
+
+type Logs struct {
+	client functionLogsClient
+	logger log.Logger
+
+	args struct {
+		NameOrID string
```
Comment (dianadoherty) [inline cmd/meroxa/root/functions/logs.go:31]: Same as meroxa-go, it's best to refer to it as UUID
```suggestion
		NameOrUUID string
```

File: cmd/meroxa/root/functions/logs.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	l.logger = logger
+}
+
+func (l *Logs) Client(client meroxa.Client) {
+	l.client = client
+}
+
+func (l *Logs) ParseArgs(args []string) error {
+	if len(args) < 1 {
+		return errors.New("requires function name")
```
Comment (dianadoherty) [inline cmd/meroxa/root/functions/logs.go:75]: Not sure if it's the standard, but what about including the uuid as well?
```suggestion
		return errors.New("requires function name or uuid")
```

File: cmd/meroxa/root/functions/logs.go [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+	"github.com/meroxa/cli/log"
+	"github.com/meroxa/meroxa-go/pkg/meroxa"
+)
+
+var (
+	_ builder.CommandWithDocs    = (*Logs)(nil)
+	_ builder.CommandWithArgs    = (*Logs)(nil)
+	_ builder.CommandWithClient  = (*Logs)(nil)
+	_ builder.CommandWithLogger  = (*Logs)(nil)
+	_ builder.CommandWithExecute = (*Logs)(nil)
```
Comment (dianadoherty) [inline cmd/meroxa/root/functions/logs.go:19]: Did you want to make this command hidden? 
```suggestion
	_ builder.CommandWithExecute = (*Logs)(nil)
	_ builder.CommandWithHidden  = (*Repair)(nil)

```

Comment (dianadoherty) [reply cmd/meroxa/root/functions/logs.go:19]: If it's not supposed to be hidden, then can you run `make build`? It's supposed to also create documentations for each command

**Review by dianadoherty — COMMENTED** [review]
~~Can you include the example of how logs look like after running this command? It makes it easier to notice if there are any displaying issues :)~~ Never mind, I read your comment 🤦

File: cmd/meroxa/root/functions/logs.go (hunk 3 above)
Comment (owenthereal) [reply cmd/meroxa/root/functions/logs.go:19]: The root command is already marked as hidden: https://github.com/meroxa/cli/blob/5590e3e3efedb8ecf1bd7ed27b510a3f14fed964/cmd/meroxa/root/functions/functions.go#L17. So `make build` won't build any docs for its subcommands

**Review by dianadoherty — APPROVED** [review]

**Pull request merged**